config/initializers/kaminari_config.rb

```diff
@@ -0,0 +1,10 @@
+Kaminari.configure do |config|
+  # config.default_per_page = 25
```

Comment:
Does it make sense for you to use a default since you're using 10 in both places? Also, DX these comments if you're not going to use them